lib/cdo/rake_utils.rb

```diff
@@ -131,6 +132,10 @@ def self.run_bundle_command(*args)
     end
   end
 
+  def self.python_venv_install
```

Comment:
This ruby method just runs `pdm install`. I was thinking this would be sort of like running `yarn install`: active JS developers probably run the command directly rather than via rake? LMK if that doesn't make sense to you, happy to add a direct rake target too.

Reply:
yes, this makes sense to leave as is then, thanks!